instrumentation/opentelemetry-instrumentation-django/src/opentelemetry/instrumentation/django/middleware.py

```diff
@@ -213,9 +203,17 @@ def process_response(self, request, response):
             )
             request.META.pop(self._environ_span_key)
 
-            request.META[self._environ_activation_key].__exit__(
-                None, None, None
-            )
+            exception = request.META.pop(self._environ_exception_key, None)
```

Comment:
Is `process_response` always called after an exception?

Reply:
Yes. Earlier we popped span context from request.META so `process_response` would essentially be a noop after an exception but Django calls it for every request.